specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/ipTags.json

```diff
@@ -0,0 +1,411 @@
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/ipTags": {
+      "get": {
+        "tags": [
+          "IpTags"
+        ],
+        "operationId": "IpTags_ListByResourceGroup",
+        "description": "Gets all IpTags in a resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ipTag resources.",
+            "schema": {
+              "$ref": "#/definitions/IpTagsListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ListByResourceGroup_IpTags": {
+            "$ref": "./examples/IpTagsListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/ipTags": {
+      "get": {
+        "tags": [
+          "IpTags"
+        ],
+        "operationId": "IpTags_List",
+        "description": "Gets all IpTags in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ipTag resources.",
+            "schema": {
+              "$ref": "#/definitions/IpTagsListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List_IpTags": {
+            "$ref": "./examples/IpTagsListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IpTagsListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpTags"
+          },
+          "description": "The list of iptag information resources."
+        }
+      },
+      "description": "Response for the ListIpTags API service call."
+    },
+    "IpTags": {
+      "properties": {
+        "provisioningState": {
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the IpTags resource."
+        },
+        "changeNumber": {
```

Comment:
is changeNumber redundant with etags? What is it's function?

Reply:
The changeNumber refers to the version of the IpTags resource visible to customer. BTW, I am looking to store an array of ""$ref": "./network.json#/definitions/SubResource" + changeNumber in the referencing object. How do I define this in Swagger?

Reply:
I am removing changeNumber in the current revision. Done.